Framework/TestHelpers/inc/MantidTestHelpers/FakeObjects.h

```diff
@@ -177,9 +177,9 @@ class WorkspaceTester : public AxeslessWorkspaceTester {
     AxeslessWorkspaceTester::init(numspec, j, k);
 
     // Put an 'empty' axis in to test the getAxis method
-    m_axes.resize(2);
-    m_axes[0] = new Mantid::API::RefAxis(j, this);
-    m_axes[1] = new Mantid::API::SpectraAxis(this);
+    m_axes.resize(0);
```

Comment:
What's the reason for choosing `push_back` over the original implementation? I would still keep the `.resize(2)` anyway to avoid additional allocations.